sci-libs/caffe2/caffe2-2.1.2-r3.ebuild

```diff
@@ -223,16 +223,14 @@ src_install() {
 	rm -rf python
 	mkdir -p python/torch/include || die
 	mv "${ED}"/usr/lib/python*/site-packages/caffe2 python/ || die
-	mv "${ED}"/usr/include/torch python/torch/include || die
 	if use cuda; then
 		mv "${ED}${S}"/nvfuser python/nvfuser || die
 		mv "${ED}"/usr/$(get_libdir)/nvfuser.so python/nvfuser/_C.so || die
 	fi
-	rm -rf "${ED}${S}"/test
-	rm -rf "${ED}${S}"/third_party
 	cp torch/version.py python/torch/ || die
 	python_domodule python/caffe2
 	python_domodule python/torch
+	dosym -r /usr/include/torch $(python_get_sitedir)/torch/include/torch # bug 923269
```

Comment:
`$(python_get_sitedir)` includes `${EPREFIX}`, so you'd get a double prefix here. You need to use `ln -s` directly, and then `${D}$(python_get_sitedir)`.

Reply:
Yep, `ln -s` works, thanks.

Reply:
Oh, I'm so sorry for missing this, but you should really use relative path for the source directory. You can rely on `$(python_get_sitedir)` returning the same directory level, so you can hardcode the right number of `../`s.

Reply:
Today I learned!